### PR TITLE
[CI Skip] Freecad occt vtk various

### PR DESCRIPTION
**Pull request by yopito**
Changes:
* `med`: store its .h files in `/usr/include/med` instead of polluting `/usr/include/`
* `occt`: fix musl patch of  `OSD_signal.cxx`: assign values to data as much as possible
*  fix check-update on various packages
* various small gliches
* labelled `[ci skip]` : this PR is too big for CI constraints. However, personal builds (including cross) and basic runtime are fine

PR move from old repo PR https://github.com/voidlinux/void-packages/pull/14002

### CODE REVIEW

**Comment by Piraty** [comment]
@yopito Nice changes 
Can you rebase ?

**Comment by yopito** [comment]
@Piraty : here it is. thanks for your kind words.

**Comment by Piraty** [comment]
everything builds fine and freecad runs fine as well (i did just very basic function checking)
+1

**Comment by Piraty** [comment]
@yopito still [hold] ?

**Comment by yopito** [comment]
Forgot it, thx

**Comment by maxice8** [comment]
Please rebase

**Comment by yopito** [comment]
I will ASAP, maybe this weekend.

**Comment by yopito** [comment]
@maxice8 : rebase done, could you review this PR and merge it if OK ?

File: srcpkgs/freecad/template [hunk 1]
```diff
@@ -27,7 +28,7 @@ depends="python-matplotlib python-pyside qt-devel-tools qt-plugin-sqlite python-
 
 pycompile_dirs="usr/lib/${pkgname}/Mod usr/lib/${pkgname}/data/Mod"
 
-short_desc="A general purpose 3D CAD modeler"
+short_desc="General purpose 3D CAD modeler"
 maintainer="yopito <pierre.bourgin@free.fr>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/freecad/template]: SPDX identifier

File: srcpkgs/occt/template [hunk 2]
```diff
@@ -11,13 +11,16 @@ makedepends="freetype-devel glu-devel freeimage-devel gl2ps-devel tbb-devel
  tcl-devel tk-devel"
 short_desc="OpenCASCADE Technology - library for CAD/CAM/CAE applications"
 maintainer="Piraty <piraty1@inbox.ru>"
-license="LGPL-2.1"
+license="LGPL-2.1-only,OpenCascade-1.0-LGPL-exception-not-SPDX"
```
Comment (maxice8) [inline srcpkgs/occt/template]: missing space between comma and start of another license.

wtf is that second thing

File: srcpkgs/freecad/template (hunk 1 above)
Comment (yopito) [reply srcpkgs/freecad/template]: fixed

File: srcpkgs/occt/template (hunk 2 above)
Comment (yopito) [reply srcpkgs/occt/template]: rechecked and fixed

**Comment by Piraty** [comment]
Looks fine to me ! thanks @yopito

**Comment by maxice8** [comment]
[ci skip] is failing, needs to be in every commit

File: srcpkgs/freecad/template [hunk 3]
```diff
@@ -15,7 +15,8 @@ _inst_prefix=/usr/lib/${pkgname}
 configure_args="
  -DBUILD_QT5=OFF
  -DCMAKE_INSTALL_PREFIX=${_inst_prefix}
- -DCMAKE_INSTALL_LIBDIR=${_inst_prefix}/lib"
+ -DCMAKE_INSTALL_LIBDIR=${_inst_prefix}/lib
+ -DMEDFILE_INCLUDE_DIRS=/usr/include/med"
```
Comment (maxice8) [inline srcpkgs/freecad/template]: should problably be ${XBPS_CROSS_BASE} so that it expands to a cross-compilation friendly path when cross building.

**Comment by yopito** [comment]
thanks for the merge.

**Pull request merged**